Gruntfile.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (grunt) {
         options: {
           html: 'public/index.html'
         },
-        src: 'design/icon.png',
+        src: 'app/assets/web-app-icon.png',
```

Comment:
Darn, I forgot to update the icon path in `Gruntfile.js`—do you still want me to do that, or is it irrelevant now that you are declaring `icon-16x16.png` obsolete?